editor/.gitignore

```diff
@@ -27,3 +27,5 @@ pom.xml.asc
 /preflight-report.txt
 /.calva
 /.lsp
+/.portal
+/defunload-issues.txt
```

Comment:
The `defunload-issues.txt` report is written to the editor logs folder, which happens to be `defold/editor` in dev builds.